src/components/ChatGhostUI/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Dimensions} from 'react-native';
+import _ from 'underscore';
+import ContentLoadingOneLine from './ContentLoadingElements/ContentLoadingOneLine';
+import ContentLoadingTwoLines from './ContentLoadingElements/ContentLoadingTwoLines';
+import ContentLoadingThreeLines from './ContentLoadingElements/ContentLoadingThreeLines';
+
+
+const ChatList = () => {
+    const windowHeight = Dimensions.get('window').height;
```

Comment:
I am shocked to see this in your PR. Please use withWindowDimensions HOC...

Reply:
What was I thinking! 😨